sharding/indexHashedNodesCoordinator.go

```diff
@@ -115,6 +115,14 @@ func NewIndexHashedNodesCoordinator(arguments ArgNodesCoordinator) (*indexHashed
 		log.Error("saving initial nodes coordinator config failed",
 			"error", err.Error())
 	}
+	log.Info("new nodes config is set for epoch", "epoch", arguments.Epoch)
+	currentNodesConfig := ihgs.nodesConfig[arguments.Epoch]
+	displayNodesConfiguration(
```

Comment:
This print should be done only if currentNodesConfig is not nil, otherwise it will panicked here

Reply:
fixed